recipes/renode-cli/helpers/renode_build_with_dotnet.ps1

```diff
@@ -0,0 +1,71 @@
+param (
+    [string]$framework_version
+)
+
+$SRC_DIR = Resolve-Path "$Env:SRC_DIR"
+
+$OUTPUT_DIRECTORY = "$SRC_DIR\output"
+$CONFIGURATION = "Release"
+$BUILD_PLATFORM = "Any CPU"
+$HEADLESS = $true
+$NET = $true
+$TFM = "net$framework_version-windows"
+$HOST_ARCH = "i386"
+$CMAKE_COMMON = ""
+
+$PARAMS = @()
+if ($Env:CC) {
+    $PARAMS += "/p:CompilerPath=$Env:CC"
+    $PARAMS += "/p:LinkerPath=$Env:CC"
+}
+if ($Env:AR) {
+    $PARAMS += "/p:ArPath=$Env:AR"
+}
+$PARAMS += $args
```

Comment:
See my previous comments about only supporting what is needed to build in conda-forge.